GitUI/UserControls/ExTextBox.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace GitUI.UserControls
+{
+    internal sealed class ExTextBox : TextBox
```

Comment:
We have an established naming scheme (for better or for worse):
```suggestion
    internal sealed class TextBoxEx : TextBox
```